script.js

```diff
@@ -0,0 +1,132 @@
+
+import { updateTime } from './Modules/clock.js';
+updateTime ();
+import * as Navbar from './Modules/nav-bar.js'
+
+const bookList = document.getElementById('book-list');
+const addBook = document.getElementById('add-book');
+const newTitle = document.getElementById('new-title');
+const newAuthor = document.getElementById('new-author');
+const allBooks = document.getElementById('all-books');
+let bookData = [];
+
+function setBorder() {
+  const data = JSON.parse(localStorage.getItem('bookData'));
+  if (bookData.length > 0 || data.length > 0) {
+    allBooks.classList.add('border');
+  } else if (bookData.length === 0 && data.length === 0) {
+    allBooks.classList.remove('border');
+  }
+}
+function setStyles() {
+  let index = 1;
+  const liList = document.querySelectorAll('.book');
+  liList.forEach((element) => {
+    if (index % 2 !== 0) {
+      element.classList.add('bookBlack');
+    }
+    index += 1;
+  });
+}
+
+class Book {
+  constructor(title = 'title', author = 'author', id = '0') {
+    this.title = title;
+    this.author = author;
+    this.id = id;
+  }
+
+  addBook() {
+    // This will add itself to the bookData Array
+    bookData.push(this);
+  }
+
+  removeBook() {
+    // This Will Remove It self From the bookData Array
+    bookData.filter((book) => book.id !== this.id);
+  }
+}
+
+// Function create the div and li for each new book
+function getLi(book) {
```

Comment:
- [ ] Kindly modify this to use ES6 syntax by changing it to `arrow function` 👍🏽